.github/ISSUE_TEMPLATE/docs_issue.md

```diff
@@ -1,31 +0,0 @@
----
```

Comment:
@fzn0x I believe that the website repo is missing issue templates. Can you add this there? Maybe with the config as well?

Reply:
Sure @WikiRik

Reply:
Track https://github.com/sequelize/website/pull/349